package.json

```
@@ -2,11 +2,21 @@
   "name": "urql",
   "version": "1.0.5",
   "description": "A highly customizable and versatile GraphQL client for React",
```

Let's also add this :tada:

```suggestion
  "homepage": "https://formidable.com/open-source/urql",
```